src/Model/Invoices/PagedResult.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace App\Model\Invoices;
+
+class PagedResult
+{
+    public array $items = [];
+    public int $startAt = 0;
+    public int $maxResults = 0;
+    public int $total = 0;
+
+    public function __construct(array $items, int $startAt, int $maxResults, int $total)
+    {
+        $this->items = $items;
+        $this->startAt = $startAt;
+        $this->maxResults = $maxResults;
+        $this->total = $total;
+    }
```

Comment:
Consider using https://www.php.net/manual/en/language.oop5.decon.php#language.oop5.decon.constructor.promotion here:
```suggestion
    public function __construct(
      public array $items,
      public int $startAt,
      public int $maxResults,
      public int $total
    )
    {
    }
```